Client/WarFare/UIItemUpgrade.h

```diff
@@ -0,0 +1,139 @@
+﻿// CUIItemUpgrade.h: interface for the CUIItemUpgrade class.
+//
+//////////////////////////////////////////////////////////////////////
+
+#if !defined(AFX_UIItemUpgrade_H__fd9f5093_0ed3_4c08_9e31_19c40773b24d__INCLUDED_)
+#define AFX_UITRANSACTIONDLG_H__fd9f5093_0ed3_4c08_9e31_19c40773b24d__INCLUDED_
+
+#if _MSC_VER > 1000
+#pragma once
+#endif // _MSC_VER > 1000
+
+#include <N3Base/N3UIBase.h>
+#include <N3Base/N3UIArea.h>
+#include <N3Base/N3UIString.h>
+#include <N3Base/N3UIImage.h>
+#include <N3Base/N3UIButton.h>
+#include "GameProcedure.h"
+
+#include "GameDef.h"
+#include "N3UIWndBase.h"
+
+
+
+//////////////////////////////////////////////////////////////////////
+
+class CUIItemUpgrade : public CN3UIWndBase
+{
+	friend class CUIInventory;
+
+public:
+	// 직접 접근해야 할 객체 참조 포인터
+	__IconItemSkill* m_pMyUpgradeSLot[MAX_ITEM_UPGRADE_SLOT]; // Upgrade and Trina Scroll Slot
+	__IconItemSkill* m_pMyUpgradeInv[MAX_ITEM_INVENTORY];
+	__IconItemSkill* m_pBackupUpgradeInv[MAX_ITEM_INVENTORY];
+	__IconItemSkill* m_pUpgradeItemSlot;	// Which item to upgrade
+	__IconItemSkill* m_pUpgradeResultSlot;
+	int8_t m_iUpgradeSlotInvPos[MAX_ITEM_UPGRADE_SLOT+1];
+	CN3UIString* m_pStrMyGold;
+	
+	// Animation state management
+	enum AnimationState {
+		ANIM_NONE = 0,
+		ANIM_START,
+		ANIM_FLIPFLOP,
+		ANIM_RESULT,
+		ANIM_COVER_OPENING,
+		ANIM_DONE
+	} m_eAnimationState = ANIM_NONE;
```

Comment:
Things are mixed up here: we're inlining initialisers here, but we're not doing so for the above.

In the interests of consistency with the rest of the codebase (and even this class itself), move these initialisers to the constructor.